### DISCUSSION TITLE
Apps can't find packages

### DISCUSSION THREAD

**Discussion by jeroenvanhattem** [vercel/turborepo#7211] [Help] [answered]
I created a Turborepo monorepo from the Vercel starter template. Then I added 2 new packages (so next to the default `ui` package) named `redux-store` and `hooks`. I deleted the `docs` app, but kept the `web` app. 

The `web` app is able to find the `ui` and `hooks` packages (doesn't use `redux-store`). Then I copied a Next.js project to the `apps` folder so it can use reusable components from the `packages` folder. This new project is able to find the `ui` package, but not `hooks` and `redux-store`.

Everything works locally. All projects are able to find everything. Building it with `turbo build` goes well. The build only fails on Vercel.

All apps have the packages listed as dependency as such: 
`
"hooks": "workspace:*",
"redux-store": "workspace:*",
"ui": "workspace:*",
`
Repository to test it out is here:
https://github.com/jeroenvanhattem/web_monorepo

Any help would be greatly appreciated! Please leave a comment if you need extra information!

I deployed it using Vercels default settings when importing the Github repository. So these are identical for the web and portal apps (other than the root directory. This points to their respective paths)
![image](https://github.com/vercel/turbo/assets/15092050/5e5ccf1a-5e87-41d3-bd3c-2cffb927a964)

My project structure is as follows:
![image](https://github.com/vercel/turbo/assets/15092050/cab72996-eb9b-469e-955e-471e18421706)

The Vercel build logs show the following error:
![image](https://github.com/vercel/turbo/assets/15092050/a6db10e7-845d-43ea-bb9e-1fd6c4a51bfd)

**Comment by anthonyshew** [comment]
I've followed a similar process just now to see if I could reproduce this and couldn't.

Do you have a reproduction repo you could share? 🙏

**Reply by jeroenvanhattem** [reply]
I created a repo for this:
https://github.com/jeroenvanhattem/web_monorepo

**Reply by anthonyshew** [reply, marked as answer]
I made you a PR here! https://github.com/jeroenvanhattem/web_monorepo/pull/1

Should help. 😄 

![CleanShot 2024-02-02 at 09 21 37@2x](https://github.com/vercel/turbo/assets/35677084/23f86412-e766-41ea-8318-e33d00ceb7f9)

**Reply by jeroenvanhattem** [reply]
Thank you so much for your effort and quick responses! I really appreciate it! I merged your PR in, but I still get the same errors in the pipeline. Are there any extra steps I have to make during deployment to Vercel? I also noticed that the PR said the build for the portal project failed. Could you maybe show how you deployed the project?

**Reply by jeroenvanhattem** [reply]
It worked! Locally I had the `store.ts` file in the `redux-store` package as `Store.ts`. I guess you changed it to `store.ts`, but Git doesn't register a file as changed if you only changed capitalization. So you have to change it to `_store.ts`, commit and change it to `store.ts`. That did the trick! Thank you much for your help and very fast replies. You're amazing and I hope you enjoy your weekend! 🚀 🍻

**Reply by anthonyshew** [reply]
Sorry about that! I forgot casing can matter on certain combinations of git + operating system.

Glad it worked. Enjoy!
